src/bin/add/main.rs

```diff
@@ -120,11 +120,11 @@ fn is_sorted(mut it: impl Iterator<Item = impl PartialOrd>) -> bool {
 }
 
 fn handle_add(args: &Args) -> Result<()> {
-    let manifest_path = if let Some(ref pkgid) = args.pkgid {
-        let pkg = manifest_from_pkgid(args.manifest_path.as_deref(), pkgid)?;
+    let manifest_path = if let Some(pkgid) = args.workspace.package.get(0) {
+        let pkg = manifest_from_pkgid(args.manifest.manifest_path.as_deref(), pkgid)?;
         Cow::Owned(Some(pkg.manifest_path.into_std_path_buf()))
     } else {
-        Cow::Borrowed(&args.manifest_path)
+        Cow::Borrowed(&args.manifest.manifest_path)
     };
```

Comment:
- This is ignore `exclude`, `all`, and `workspace` arguments
- This is ignoring all other `package` values

If it isn't mapping well to `add`, its ok if we don't use `clap_cargo`

Reply:
@epage I think adding the same dependency to multiple crates in a workspace could be useful. I've made it`package.get(0)` just to make the code compile. It will be changed to processing the whole vector.

Reply:
It might be worth breaking that out into its own PR, after this one is done, so we can focus on one user-facing aspect at a time (existing multi-crate behavior vs making another command multi-crate)